### PR TITLE
Add missing space around cast and variable

### PR DESCRIPTION
**Pull request by MathiasReker**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | /
| Type?         | improvement
| Category?     | CO
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | /
| How to test? | /

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11818)

### CODE REVIEW

File: admin-dev/functions.php [hunk 1]
```diff
@@ -453,7 +453,7 @@ function runAdminTab($tab, $ajax_mode = false)
             if (!$ajax_mode && Shop::isFeatureActive() && Shop::getContext() != Shop::CONTEXT_ALL && Context::getContext()->controller->multishop_context != Shop::CONTEXT_ALL) {
                 echo '<div class="multishop_info">';
                 if (Shop::getContext() == Shop::CONTEXT_GROUP) {
-                    $shop_group = new ShopGroup((int)Shop::getContextShopGroupID());
+                    $shop_group = new ShopGroup((int) Shop::getContextShopGroupID());
                     printf(Translate::getAdminTranslation('You are configuring your store for group shop %s'), '<b>'.$shop_group->name.'</b>');
```
Comment (LouiseBonnard) [inline admin-dev/functions.php]: Is it localized?

File: admin-dev/functions.php [hunk 2]
```diff
@@ -453,7 +453,7 @@ function runAdminTab($tab, $ajax_mode = false)
...
                 } elseif (Shop::getContext() == Shop::CONTEXT_SHOP) {
                     printf(Translate::getAdminTranslation('You are configuring your store for shop %s'), '<b>'.Context::getContext()->shop->name.'</b>');
```
Comment (LouiseBonnard) [inline admin-dev/functions.php]: Same question here.

File: install-dev/upgrade/php/add_new_status_stock.php [hunk 3]
```diff
@@ -41,11 +41,11 @@ function add_new_status_stock()
...
-        $lastIdTab = (int)Db::getInstance()->Insert_ID();
+        $lastIdTab = (int) Db::getInstance()->Insert_ID();
 
         // ps_tab_lang
         foreach ($languages as $lang) {
-            $idLang    = (int)$lang['id_lang'];
+            $idLang    = (int) $lang['id_lang'];
             $stockName = pSQL(
                 $translator->trans(
                     'Stock',
```
Comment (LouiseBonnard) [inline install-dev/upgrade/php/add_new_status_stock.php]: ... and here!

**Comment by PierreRambaud** [comment]
@LouiseBonnard No wording needed :)

**Pull request closed without merging**